solidity/app/cl_utils.js

```diff
@@ -0,0 +1,33 @@
+const Eth = require('ethjs')
+
+
+clUtils = {
+  personalAccount: '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f',
```

Comment:
Should this be checked in or a parameter?

Reply:
Probably a good idea to make it overridable via config, but that is probably the default we want. It's an address hardcoded into devnet/gethnet.